tools/example-module/generate-example-module.ts

```diff
@@ -44,8 +44,8 @@ function inlineExampleModuleTemplate(parsedData: ExampleMetadata[]): string {
 
   return fs.readFileSync(require.resolve('./example-module.template'), 'utf8')
     .replace('${exampleImports}', exampleImports)
-    .replace('${exampleComponents}', JSON.stringify(exampleComponents))
-    .replace('${exampleList}', `[${exampleList}]`)
+    .replace('${exampleComponents}', JSON.stringify(exampleComponents, null, 2))
+    .replace('${exampleList}', `[\n  ${exampleList.join(',\n  ')}\n]`)
```

Comment:
FYI: we could also use `JSON.stringify(X, null, 2)`, but we don't want the elements to be quoted and just joining the lines properly is faster.